xarray/backends/zarr.py

```diff
@@ -430,26 +454,24 @@ def get_variables(self):
         )
 
     def get_attrs(self):
-        return dict(self.zarr_group.attrs.asdict())
+        return {
+            k: v
+            for k, v in self.zarr_group.attrs.asdict().items()
+            if not k.startswith("_NC")
+        }
 
```

Comment:
It feels a little inconsistent to me to always filter attributes with the `_NC` prefix from groups, but not always from arrays (only `if try_nczarr`).

My inclination is to _always_ filter both attributes, even if not checking for NCZarr dimensions. My guess is that users don't really want to see them, if they are normally hidden by the netCDF library.

Reply:
Right, I wasn't too happy about it either. `"_NC"` attributes are always either hidden or dropped now.